code/core/src/manager/components/layout/Layout.tsx

```diff
@@ -108,8 +113,10 @@ const useLayoutSyncingState = ({
     ? internalDraggingSizeState
     : managerLayoutState;
 
+  const customisedNavSize = api.getNavSizeWithCustomisations(navSize);
```

Comment:
I don't understand why this fails in CI, as I've added the signature to the SubAPI for layout. Any advice on what I did wrong?